doc/classes/KinematicCollision2D.xml

```diff
@@ -32,7 +32,7 @@
 		<method name="get_collider_rid" qualifiers="const">
 			<return type="RID" />
 			<description>
-				Returns the colliding body's [RID] used by the [PhysicsServer2D].
+				Returns the [RID] of the colliding body in the [PhysicsServer2D].
```

Comment:
Why the change? Although a bit out of date, [the guidelines prefer `'s` whenever it makes sense](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#use-s-to-show-possession).